contracts/staking/contracts/test/TestStaking.sol

```diff
@@ -0,0 +1,53 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "../src/Staking.sol";
+
+
+contract TestStaking is
+    Staking
+{
+    // Stub out `payProtocolFee` to be the naive payProtocolFee function so that tests will
```

Comment:
What are your thoughts on creating some internal getter function for `WETH_ASSET_DATA` (`getWethAssetData`), using that in the actual `payProtocolFee`, and then stubbing it out in this contract? That way we can test with actual WETH transfers. It's probably a couple extra gas but the tradeoff feels worth it to me.

Reply:
This could actually be a cool pattern to use for all deployment constants. Not sure if we have any similar situations with any of the other constants at the moment.

Reply:
As discussed offline, I'm a big fan and I think that we should do it on an as-needed basis.

Reply:
Since the integration tests aren't being implemented in this PR, I'm going to hold off on this until it's needed.